content/en/blog/_posts/2020-03-16-kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blog
+title: SIG Scalability - Learning Kubernetes the Hard Way
+date: 2020-03-16
+slug: sig-scalability-learning-kubernetes-the-hard-way
+---
+
+**Authors:** Alex Handy
+
+There is a strain of thinking in the software development community that pushes for the most challenging and rigorous possible method of learning a new language or system. These tend to go by the moniker of "Learn \_\_ the Hard Way." Examples abound: Learn Code the Hard Way, Learn Python the Hard Way, and of course, the one that started the whole movement, Zed Shaw's courses in the topic.
+
+While there are folks out there who offer you a "Learn Kubernetes the Hard Way" type experience (most notably Google's Kelsey Hightower), any "Hard Way" project should attempt to cover every aspect of the core topic's principles.
+
+Therefore, the real way to "Learn Kubernetes the Hard Way," is to join the CNCF and get involved in the project itself. And there is only one SIG that could genuinely offer a full-stack learning experience for Kubernetes: SIG Scalability.
+
+The team behind SIG Scalability is responsible for detecting and dealing with issues that arise when Kubernetes clusters are working with upwards of a thousand nodes. Said Wojiciech Tyczynski, a staff software engineer at Google and a member of SIG Scalability, the standard size for a test cluster for this SIG is over 5,000 nodes.
+
+And yet, this SIG is not composed of Ph.D.'s in highly scalable systems designs. Many of the folks working with Tyczynski, for example, joined the SIG knowing very little about these types of issues, and often, very little about Kubernetes.
+
+Working on SIG Scalability is like jumping into the deep end of the pool to learn to swim, and the SIG is inherently concerned with the entire Kubernetes project. SIG Scalability focuses on how Kubernetes functions as a whole and at scale. The SIG Scalability team members have an impetus to learn about every system and to understand how all systems interact with one another.
+
```

Comment:
```suggestion

## A complex and rewarding contributor experience
```

I'm looking to transition this into the next part of the narrative that calls to the reader we want to attract. "Hey, this is hard, and if you like that, then it's for you!" Feel free to adjust.